.talismanrc

```diff
@@ -0,0 +1,45 @@
+fileignoreconfig:
```

Comment:
Phil had a suggested workflow where we don't commit these. 

 I think it made sense, because if these files are changed, the checksum will be different, so Talisman will flag them again. So you don't gain anything from committing this. The alternative is to review the things it flags, and then ignore it locally using git commit --no-verify. 

I will try to find his post though.

Reply:
Hmm I think probably post was too long ago for Slack to have it.

Reply:
Ahhh, no, my fault. I was aware of that proposal but added this accidentally. Will fix.

Reply:
Ahh, looks like I added it to git ignore, but forgot to remove it from the commits